README.md

```diff
@@ -36,6 +36,15 @@ class Controller {
     public function get(#[Parameter("Parameter description")] int $id): JsonResponse {
         // ...
     }
+    
+    #[
+        DynamicBuilder(MyFrameworkResolver::class),
+        Property(PropertyType::REF, "prop4", ref: RefSchema::class)
```

Comment:
I left some properties as an example that DynamicBuilder can be used together with other attributes.